packages/react/src/drawer/docs/overview.mdx

```diff
@@ -411,3 +411,62 @@ When a drawer closes, focus is returned to the element that opened it. You can o
 	);
 };
 ```
+
+## Using a form in the drawer
+
+To create an accessible form within a drawer we need to do two things:
+
+1. Add an `id` to the `<form>`.
+2. Pass that `id` to the primary Button's `form` prop along with `type="submit"`.
+
+This will allow keyboard users to submit the form by pressing "enter" on the form inputs.
+
+```jsx live
+() => {
+	const formRef = React.useRef(null);
+	const [isDrawerOpen, openDrawer, closeDrawer] = useTernaryState(false);
+	const onSubmitForm = (event) => {
+		event.preventDefault();
+
+		const formData = Object.fromEntries(new FormData(formRef.current));
+		console.log('Form submitted with values:', formData);
+
+		closeDrawer();
+	};
+
+	return (
+		<React.Fragment>
+			<Button onClick={openDrawer}>Open Drawer</Button>
+
+			<Drawer
+				actions={
+					<ButtonGroup>
+						<Button type="submit" form="form-id" onClick={onSubmitForm}>
+							Submit form
+						</Button>
+
+						<Button variant="tertiary" onClick={closeDrawer}>
+							Cancel
+						</Button>
+					</ButtonGroup>
+				}
+				isOpen={isDrawerOpen}
+				onClose={closeDrawer}
+				title="Example form"
+			>
+				<form aria-label="Example form in drawer" id="form-id" ref={formRef}>
```

Comment:
I don't know if we need this `aria-label` here (and in Storybook).

When you open a dialog, the title is read out because it's focused. This label is additionally read out the first time you land on an input (and no subsequent inputs) or if you use the landmarks. There are no other landmarks on the page when a drawer is open, because everything else is hidden, so it feels weird to have 1 lone landmark in a drawer

Reply:
No worries, sounds good to me.

```suggestion
				<form id="form-id" ref={formRef}>
```